src/MatdaAIga.LinkConverter/Models/LinkModel.cs

```diff
@@ -0,0 +1,41 @@
+using System.Text.Json.Serialization;
+
+namespace MatdaAIga.LinkConverter.Models;
+
+/// <summary>
+/// Description of markdown file with a list of <see cref="LinkItem"/> objects
```

Comment:
`/// This represents the collection entity of the links rendered in the links page.`